packages/text-area/src/TextArea.js

```diff
@@ -25,28 +25,52 @@ export default class TextArea extends Component {
     /**
      * The visual variant of the textarea
      */
-    variant: PropTypes.oneOf(variantTypes)
+    variant: PropTypes.oneOf(variantTypes),
+    /**
+     * Adds custom/overriding styles
+     */
+    stylesheet: PropTypes.func
   };
 
   static defaultProps = {
     variant: "line"
   };
 
+  // constructor(props) {
```

Comment:
Just looked at it again and either way it's pretty ugly, so if you prefer your original way that's fine too, but if not we should remove the comments.